File: src/main/java/post/Box5.java
package post;

public class Box5 extends Box {
    static float length = 39.5f, width = 27.5f, height = 23f;

    public Box5() {
        super();
    }

    @Override
    public boolean validate(float length, float width, int height) {
        return !(this.length < length) && !(this.width < width) && !(this.height < height);

    }
}

File: src/main/kotlin/post/Box5.kt
package post

class Box5: Box() {
    companion object {
        @JvmStatic
        var boxLength: Float = 39.5f
        var boxWidth: Float = 27.5f
        var boxHeight: Int = 23
    }

    override fun validate(length: Float, width: Float, height: Int): Boolean =
        !(boxLength < length || boxWidth < width || boxHeight < height)
}